static/js/components/forms/BulkEnrollmentForm_test.js

```diff
@@ -91,4 +98,30 @@ describe("BulkEnrollment", () => {
     assert.isTrue(findFormikFieldByName(form, "coupon_payment_id").exists())
     assert.isTrue(form.find("button[type='submit']").exists())
   })
+
+  it("sorts products by readable_id", () => {
+    const wrapper = renderForm()
+    const form = wrapper.find("Formik")
+    const options = form.find("select[name='product_id']").props().children
+    const initialProductType = PRODUCT_TYPE_COURSERUN
```

Comment:
here, we are only testing for course runs.

Reply:
done with following test cases

1. Changing Product Id
2. Changing Product Type

- [ ]